### PR TITLE
[Merged by Bors] - Correct typos book

### PR DESCRIPTION
**Pull request by emhane**
## Issue Addressed

No issue

## Proposed Changes

Correct typos in book

## Additional Info

Nothing to add

### CODE REVIEW

File: book/src/suggested-fee-recipient.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ There is no guarantee that an execution node will use the `suggested_fee_recipie
 it may use any address it chooses. It is assumed that an honest execution node *will* use the
 `suggested_fee_recipient`, but users should note this trust assumption.
 
-The `suggested_fee_recipient` can be provided to the VC, who will transmit it to the BN. The also BN
+The `suggested_fee_recipient` can be provided to the VC, who will transmit it to the BN. Then also BN
```
Comment (macladson) [inline book/src/suggested-fee-recipient.md]: I would probably say:
```suggestion
The `suggested_fee_recipient` can be provided to the VC, who will transmit it to the BN. The BN also
```

**Review by macladson — CHANGES_REQUESTED** [review]
Just had one little comment, but otherwise looks good :+1:

**Review by macladson — APPROVED** [review]
Perfect!

**Review by michaelsproul — APPROVED** [review]
Awesome, thanks!

bors r+

**Comment by michaelsproul** [comment]
briefly cancelling this so I can include it in a batch with #3056

bors r-

**Pull request closed without merging**